dxr/templates/layout.html

```diff
@@ -49,20 +49,25 @@
                           </button>
                         </section>
                         <div class="sf-select-options sf-modal" aria-expanded="false">
-                          <ul class="selector-options" tabindex="-1">
+                          <table class="selector-options" tabindex="-1">
                             {% for filter in filters %}
-                              <li>
-                                <a href="javascript:void(0)" data-value="{{ filter.name }}:">
-                                  <span class="selector-option-label">
-                                    {{ filter.name }}
-                                  </span>
-                                  <span class="selector-option-description">
-                                    {{ filter.description }}
-                                  </span>
-                                </a>
-                              </li>
+                              <tr data-value="{{ filter.name }}:">
+                                <td class="selector-option-label">
```

Comment:
Now the middle column needs a valign: 
![screen shot 2016-07-26 at 16 59 10](https://cloud.githubusercontent.com/assets/295816/17155248/4b48b614-5352-11e6-9657-cb612b88bdcf.png)